content/en/metrics/distributions.md

```diff
@@ -14,26 +14,78 @@ further_reading:
+
+  * **Threshold Queries for SLOs**:
+  
+    _“I’d like to define a 30-day SLO where the p95 of request latency is < 5 seconds”_
 
-* **Calculation of percentile aggregations**: Percentile aggregations (p50, p75, p90, p95, p99) are calculated from the raw data across all hosts, and are therefore globally accurate.
 
 * **Customization of tagging**: This functionality allows you to control the tagging scheme for custom metrics for which host-level granularity is not necessary (for example, transactions per second for a checkout service).
 
 See the [Developer Tools section][1] for more implementation details. 
 
 **Note:** Because distributions are a new metric type, they should be instrumented under new metric names during submission to Datadog.
 
-## Aggregations
+## Enabling percentile aggregations
+
+Like other metric types, such as `gauges` or `histograms`, distributions have the following aggregations available: `count`, `min`, `max`, `sum`, and `avg`. Distributions are initially tagged the same way as other metrics, with custom tags set in code. They are then resolved to host tags based on the host that reported the metric. 
 
-Like other metric types, such as `gauges` or `histograms`, distributions have the following aggregations available: `count`, `min`, `max`, `sum`, and `avg`. Distributions are initially tagged the same way as other metrics, with custom tags set in code. They are then resolved to any host tag based on the host that reported the metric. You can also calculate percentile aggregations for all queryable tags on your distribution, specified on the [Metrics Summary][2] page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99`.
+However, you can additionally enable the calculation of globally accurate percentile aggregations for all queryable tags on your distribution on the Metrics Summary page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99` as well as advanced query options like threshold queries for SLOs.
 
 {{< img src="metrics/distributions/percentiles.mp4" alt="Enable Percentiles" video=true style="width:80%;">}}
 
 After electing to apply percentile aggregations on a distribution metric, these aggregations are automatically available in the graphing UI:
 
 {{< img src="metrics/distributions/graph_percentiles.jpg" alt="Distribution metric aggregations" style="width:80%;">}}
 
+You can use percentile aggregations in a variety of other widgets and for alerting: 
+* **Single percentile value over any timeframe**
+
+_“What has the p95 request duration for my application been over the past week?”_ 
+
+{{< img src="metrics/distributions/percentile_QVW.jpg" alt="A query value widget displaying a single value (7.33s) for the p95 aggregation of a single metric" style="width:80%;">}}
+
+* **Percentile thresholds on metric monitors**
+_ “Alert me when the p99 of my application’s request latency is greater than 200 ms for the last 5 min.”_ 
+
```

Comment:
```suggestion
* **Percentile thresholds on metric monitors**
  _ “Alert me when the p99 of my application’s request latency is greater than 200 ms for the last 5 min.”_ 

```

Reply:
These are still displaying wrong on the staging preview, hopefully this fixes it 🤞🏻